docs/src/development/ssh.md

```diff
@@ -2,131 +2,118 @@
 
-Automation tools may also use an [API Token](/development/cli/api-tokens.md).
+SSH supports authentication with certificates or key pairs. Certificates are faster to set up and generally easier to use, provided you have a web browser available. They are also required for the Platform.sh CLI (unless using an [API token](/development/cli/api-tokens.md)) and when your organization has multifactor authentication set up.
 
-## Certificate-based authentication
+You can use key pairs to access deployed environments if you are setting up an automation tool or prefer that method.
 
-To connect using certificate-based authentication, install the [Platform.sh CLI](/development/cli/_index.md).
+Automation tools may also use [API tokens](/development/cli/api-tokens.md).
 
-Once installed, you may run `platform login` or any CLI command that would require authentication.  In either case, a browser window will open and ask you to login with your Platform.sh account credentials.  This web page is already encrypted with TLS over HTTP, making it secure.
+## Authenticate with certificates
 
-The login process will issue a certificate that gets stored in your local SSH configuration.  The certificate is automatically cycled every hour for a new certificate as long as your session is active.  If you are inactive for an extended period your certificate will expire, and the system will ask you to login again the next time you use a command that requires authentication.
+To connect using certificates:
 
-## Keypair-based authentication
+1. Install the [Platform.sh CLI](/development/cli/_index.md).
+2. Run `platform login`.
+4. In the open browser window, log in with your Platform.sh account credentials. (This webpage is encrypted with HTTPS [HTTP over TLS], making it secure.)
+5. Authorize the CLI to use your account.
 
-This process requires two [RSA keys](https://en.wikipedia.org/wiki/RSA_%28cryptosystem%29):
+A certificate gets stored in your local SSH configuration. The certificate is automatically cycled every hour for a new certificate as long as your session is active.
 
-* A **private key** kept secret by the user
-* A **public key** stored within the Platform.sh account
+If you are inactive for an extended period your certificate expires and you are asked to login again the next time you use a command that requires authentication.
 
-These keys are called the *public-private keypair* and usually look like random lines of characters, like this:
+## Authenticate with a key pair
 
-*A private key*:
+This process requires two keys:
+
+* A **private key** you must keep _secret_
+* A **public key** stored in your Platform.sh account
+
+These keys usually look like random lines of characters, like this example of [RSA keys](https://en.wikipedia.org/wiki/RSA_%28cryptosystem%29):
+
+A private key:
```

Comment:
Do we need the example of the private key here? This would generally be stored locally and we don't need access to it either, so it might not be necessary?

Reply:
I kept it because it was here and because it seemed to be about explaining the method behind the process. But it's probably unnecessary (I wouldn't have added it if it hadn't been there).